app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,85 @@
+#modalOverlay {
+  background-color: #ffffff;
+  display: none; 
+  height: 100%;
+  left: 0;
+  margin: 0;
+  opacity: 0.7;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 9999;
+}
+
+#modal {
+  background-color: #ffffff;
+  border: 2px solid #d9d4d9;
```

Comment:
Color literals like `#d9d4d9` should only be used in variable declarations; they should be referred to via variable everywhere else.